rca/account_management/views.py

```diff
@@ -0,0 +1,173 @@
+        student_user = User.objects.create_user(
+            username=form.cleaned_data["username"],
+            email=form.cleaned_data["email"],
+            first_name=form.cleaned_data["first_name"],
+            last_name=form.cleaned_data["last_name"],
+        )
+        # Add the student group
+        student_user.groups.add(student_group)
+        student_user.set_unusable_password()
+        student_user.save()
+
+        # return here if we aren't creating a page for the student
+        if not form.cleaned_data["create_student_page"]:
+            messages.success(
+                self.request,
+                f"The Student account for {student_user} has been created.",
+            )
+            return data
+
+        # Is there a student index page we can create a student page under?
+        student_index = StudentIndexPage.objects.first()
+        if not student_index:
+            messages.warning(
+                self.request,
+                f"Failed to create a Student Page matching for the new user: {student_user}, "
+                "a Parent Student Index Page is required.",
+            )
+            logger.info(
+                "There is not parent Student Index page to create a Student Page under, "
+                f"so one has not been created for user: {student_user}"
+            )
+            return data
+
+        # If a student page already exist for this user, we don't want to create one.
+        student_page = StudentPage.objects.filter(
+            first_name=form.cleaned_data["first_name"],
+            last_name=form.cleaned_data["last_name"],
+        ).exists()
+
+        if student_page:
+            messages.success(
+                self.request,
+                f"The Student account for {student_user} has been created.",
+            )
+            messages.warning(
+                self.request,
+                f"Failed to create a Student Page matching for the new user: {student_user}, "
+                "a Student Page already already exists",
+            )
+            logger.info(
+                "A Student User has been created, but a StudentPage matching "
+                f"the users name already exists: {student_page}, so one has not been created"
+            )
+        else:
+            student_index.add_child(
+                instance=StudentPage(
+                    first_name=form.cleaned_data["first_name"],
+                    last_name=form.cleaned_data["last_name"],
+                    title=f"{form.cleaned_data['first_name']} {form.cleaned_data['last_name']}",
+                    student_user_account=student_user,
+                    live=False,
+                )
+            )
+
+            # Generate a password reset link to be emailed to the user.
+            password_reset_url = self.request.build_absolute_uri(
+                get_set_password_url(student_user)
+            )
+            # Notify the user of the account.
```

Comment:
AC: When a new student account is created an email notification is sent to the user asking them to reset their password